src/main/java/com/example/firma/MailController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.firma;
+
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Component
+public class MailController {
+
+    private JavaMailSender mailSender;
+
+    public MailController(JavaMailSender mailSender) {
+        this.mailSender = mailSender;
+    }
+
+    public void sendMessageWithAttachment(String cc, String subject, String text) {
```

Comment:
Nazwa metody wskazuje na to, że dodawany jest jakiś załącznik do maila, a tak nie jest. Nazwa metody do poprawy